Multiple errors in Sawtooth simple network example

## 1. Contract src path error in Dockerfile
The following paths are incorrect in the Dockerfile, resulting in a build failure:

https://github.com/hyperledger/caliper-benchmarks/blob/122b1c47e59ba16d4436cb5e44387c3c7561b5d7/networks/sawtooth/simplenetwork/Dockerfile#L55-L57

The build error:
```
ERROR: for simple-tp-python  Cannot start service simple-tp-python: OCI runtime create failed: container_linux.go:349: starting container process caused "exec: \"simple-tp-python\": executable file not found in $PATH": unknown
```

Since the benchmark repo root is mounted as the `/project/` dir in the container, the correct path should be `/project/src/sawtooth/scenario/simple/simple_python` (so the problem is the missing `scenario` path segment).

## 2. Contract src path error in network configuration
The same applies to the corresponding network configuration file:

https://github.com/hyperledger/caliper-benchmarks/blob/122b1c47e59ba16d4436cb5e44387c3c7561b5d7/networks/sawtooth/simplenetwork/sawtooth.json#L10-L12

Resulting in:
```
2020.04.08-12:55:00.968 error [caliper] [sawtooth.js] 	invokeSmartContract failed, Error: Unable to load batch builder for simple[1.0] at src/sawtooth/simple/SimpleBatchBuilder.js::Cannot find module '/home/klenik/projects/caliper-benchmarks/src/sawtooth/simple/SimpleBatchBuilder.js'
```

## 3. Incorrect package name error
The required Sawtooth package name is also incorrect in the `SimpleBatchBuilder.js` module:

https://github.com/hyperledger/caliper-benchmarks/blob/122b1c47e59ba16d4436cb5e44387c3c7561b5d7/src/sawtooth/scenario/simple/SimpleBatchBuilder.js#L17

Resulting in:
```
2020.04.08-13:02:57.014 error [caliper] [sawtooth.js] 	invokeSmartContract failed, Error: Unable to load batch builder for simple[1.0] at src/sawtooth/scenario/simple/SimpleBatchBuilder.js::Cannot find module 'caliper-sawtooth'
```

## 4. Missing dependency in batch builder
The `cbor` dependency is required, but not installed automatically:

https://github.com/hyperledger/caliper-benchmarks/blob/122b1c47e59ba16d4436cb5e44387c3c7561b5d7/src/sawtooth/scenario/simple/SimpleBatchBuilder.js#L55

The dependency could be added to the start script.

Resulting error:
```
2020.04.08-13:08:07.241 error [caliper] [sawtooth.js] 	invokeSmartContract failed, Error: Cannot find module 'cbor'
```

## 5. Incompatible batch builder with transfer function
The following place has hard-coded invoke argument names, which do not match the `account1` and `account2` attributes of the `transfer` arguments:

https://github.com/hyperledger/caliper-benchmarks/blob/122b1c47e59ba16d4436cb5e44387c3c7561b5d7/src/sawtooth/scenario/simple/SimpleBatchBuilder.js#L51

--- a/.gitignore
+++ b/.gitignore
@@ -1,4 +1,5 @@
 # Git Ignore file
+.idea/
 launch.json
 
 # No logs
@@ -23,3 +24,5 @@ networks/fabric/config_kafka/crypto-config
 networks/fabric/config_raft/crypto-config
 networks/fabric/config_solo/crypto-config
 networks/fabric/config_solo_raft/crypto-config
+
+**/__pycache__/**

--- a/benchmarks/scenario/simple/config-sawtooth.yaml
+++ b/benchmarks/scenario/simple/config-sawtooth.yaml
@@ -1,0 +1,35 @@
+---
+# NOTE: the transfer function logic is not compatible with the corresponding Sawtooth batch builder
+test:
+  name: simple
+  description: This is an example benchmark for caliper, to test the backend DLT's
+    performance with simple account opening & querying transactions
+  workers:
+    type: local
+    number: 1
+  rounds:
+  - label: open
+    description: Test description for the opening of an account through the deployed chaincode
+    txNumber: 100
+    rateControl:
+      type: fixed-rate
+      opts:
+        tps: 50
+    arguments:
+      money: 10000
+    callback: benchmarks/scenario/simple/open.js
+  - label: query
+    description: Test description for the query performance of the deployed chaincode
+    txNumber: 100
+    rateControl:
+      type: fixed-rate
+      opts:
+        tps: 100
+    callback: benchmarks/scenario/simple/query.js
+monitor:
+  type:
+  - docker
+  docker:
+    name:
+    - all
+  interval: 1

--- a/networks/sawtooth/simplenetwork/Dockerfile
+++ b/networks/sawtooth/simplenetwork/Dockerfile
@@ -52,9 +52,9 @@ RUN pip3 install grpcio \
  && apt-get clean \
  && rm -rf /var/lib/apt/lists/*
 
-WORKDIR /project/src/sawtooth/simple/simple_python
+WORKDIR /project/src/sawtooth/scenario/simple/simple_python
 
-ENV PATH "$PATH:/project/:/project/src/sawtooth/simple/simple_python:."
+ENV PATH "$PATH:/project/:/project/src/sawtooth/scenario/simple/simple_python:."
 
 RUN echo "$PATH"
 RUN echo "`pwd`"

--- a/networks/sawtooth/simplenetwork/sawtooth.json
+++ b/networks/sawtooth/simplenetwork/sawtooth.json
@@ -2,14 +2,14 @@
   "caliper":{
     "blockchain": "sawtooth",
     "command" : {
-      "start": "docker-compose -f networks/sawtooth/simplenetwork/sawtooth-simple.yaml up -d",
+      "start": "npm i --no-save cbor && docker-compose -f networks/sawtooth/simplenetwork/sawtooth-simple.yaml up -d",
       "end" : "docker-compose -f networks/sawtooth/simplenetwork/sawtooth-simple.yaml down;(test -z \"$(docker ps -aq)\") || docker rm $(docker ps -aq)"
     }
   },
   "sawtooth": {
     "batchBuilders": {
       "simple": {
-        "1.0": "src/sawtooth/simple/SimpleBatchBuilder.js"
+        "1.0": "src/sawtooth/scenario/simple/SimpleBatchBuilder.js"
       }
     },
     "network": {

--- a/src/sawtooth/scenario/simple/SimpleBatchBuilder.js
+++ b/src/sawtooth/scenario/simple/SimpleBatchBuilder.js
@@ -14,10 +14,11 @@
 
 'use strict';
 
-const BatchBuilder = require('caliper-sawtooth').BatchBuilder;
+const BatchBuilder = require('@hyperledger/caliper-sawtooth').BatchBuilder;
 const {createHash} = require('crypto');
 const {createContext, CryptoFactory} = require('sawtooth-sdk/signing');
 const {protobuf} = require('sawtooth-sdk');
+const cbor = require('cbor');
 
 /**
  * BatchBuilder for simple use case and it use case specific logic to
@@ -52,7 +53,6 @@ class SimpleBatchBuilder extends BatchBuilder {
             const address = this.calculateAddress(addr);
             const addresses = [address];
 
-            const cbor = require('cbor');
             const payloadBytes = cbor.encode(args[i]);
 
             const transactionHeaderBytes = protobuf.TransactionHeader.encode({

